2013-12-26 00:21:57,566 - bbs_logger - ERROR - getCommentInfo() takes exactly 1 argument (2 given)
Traceback (most recent call last):
  File "bbs_snw.py", line 120, in main
    obj.main()
  File "/home/lgy/Code/lgy_git/Python/newspider/BaseBBS.py", line 46, in main
    count=self.searchWrapper(count)
  File "/home/lgy/Code/lgy_git/Python/newspider/BaseBBS.py", line 66, in searchWrapper
    self.search4EachItem(items)
  File "/home/lgy/Code/lgy_git/Python/newspider/BaseBBS.py", line 99, in search4EachItem
    self.itemProcess(item)
  File "bbs_snw.py", line 73, in itemProcess
    commentCount,readCount = self.getCommentInfo(countinfo)
TypeError: getCommentInfo() takes exactly 1 argument (2 given)
2013-12-26 00:25:04,737 - bbs_logger - ERROR - getCommentInfo() takes exactly 1 argument (2 given)
Traceback (most recent call last):
  File "bbs_snw.py", line 120, in main
    obj.main()
  File "/home/lgy/Code/lgy_git/Python/newspider/BaseBBS.py", line 46, in main
    count=self.searchWrapper(count)
  File "/home/lgy/Code/lgy_git/Python/newspider/BaseBBS.py", line 66, in searchWrapper
    self.search4EachItem(items)
  File "/home/lgy/Code/lgy_git/Python/newspider/BaseBBS.py", line 99, in search4EachItem
    self.itemProcess(item)
  File "bbs_snw.py", line 73, in itemProcess
    commentCount,readCount = self.getCommentInfo(countinfo)
TypeError: getCommentInfo() takes exactly 1 argument (2 given)
2013-12-26 00:26:14,210 - bbs_logger - ERROR - no such group
Traceback (most recent call last):
  File "bbs_snw.py", line 120, in main
    obj.main()
  File "/home/lgy/Code/lgy_git/Python/newspider/BaseBBS.py", line 46, in main
    count=self.searchWrapper(count)
  File "/home/lgy/Code/lgy_git/Python/newspider/BaseBBS.py", line 66, in searchWrapper
    self.search4EachItem(items)
  File "/home/lgy/Code/lgy_git/Python/newspider/BaseBBS.py", line 99, in search4EachItem
    self.itemProcess(item)
  File "bbs_snw.py", line 73, in itemProcess
    commentCount,readCount = self.getCommentInfo(countinfo)
  File "bbs_snw.py", line 99, in getCommentInfo
    return m.group(1),m.group(2)
IndexError: no such group
2013-12-26 00:34:09,700 - bbs_logger - ERROR - 'NoneType' object has no attribute 'group'
Traceback (most recent call last):
  File "bbs_snw.py", line 121, in main
    obj.main()
  File "/home/lgy/Code/lgy_git/Python/newspider/BaseBBS.py", line 46, in main
    count=self.searchWrapper(count)
  File "/home/lgy/Code/lgy_git/Python/newspider/BaseBBS.py", line 66, in searchWrapper
    self.search4EachItem(items)
  File "/home/lgy/Code/lgy_git/Python/newspider/BaseBBS.py", line 99, in search4EachItem
    self.itemProcess(item)
  File "bbs_snw.py", line 82, in itemProcess
    createdAt = self.convertTime(userInfoTag.text[0:userInfoTag.text.find(" -")])
  File "bbs_snw.py", line 102, in convertTime
    if m.group()!=None:
AttributeError: 'NoneType' object has no attribute 'group'
2013-12-26 09:51:17,443 - bbs_logger - ERROR - <urlopen error [Errno 111] Connection refused>
Traceback (most recent call last):
  File "bbs_a028.py", line 59, in main
    obj.main()
  File "/home/lgy/Code/lgy_git/Python/newspider/BaseBBS.py", line 46, in main
    count=self.searchWrapper(count)
  File "/home/lgy/Code/lgy_git/Python/newspider/BaseBBS.py", line 64, in searchWrapper
    items = self.nextPage(keyword)
  File "bbs_a028.py", line 17, in nextPage
    response = urllib2.urlopen(url)
  File "/usr/lib/python2.7/urllib2.py", line 126, in urlopen
    return _opener.open(url, data, timeout)
  File "/usr/lib/python2.7/urllib2.py", line 400, in open
    response = self._open(req, data)
  File "/usr/lib/python2.7/urllib2.py", line 418, in _open
    '_open', req)
  File "/usr/lib/python2.7/urllib2.py", line 378, in _call_chain
    result = func(*args)
  File "/usr/lib/python2.7/urllib2.py", line 1207, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "/usr/lib/python2.7/urllib2.py", line 1177, in do_open
    raise URLError(err)
URLError: <urlopen error [Errno 111] Connection refused>
2013-12-26 10:13:26,885 - bbs_logger - ERROR - global name 'keyword' is not defined
Traceback (most recent call last):
  File "bbs_a028.py", line 91, in main
    obj.main()
  File "/home/lgy/Code/lgy_git/Python/newspider/BaseBBS.py", line 46, in main
    count=self.searchWrapper(count)
  File "/home/lgy/Code/lgy_git/Python/newspider/BaseBBS.py", line 66, in searchWrapper
    self.search4EachItem(items)
  File "/home/lgy/Code/lgy_git/Python/newspider/BaseBBS.py", line 99, in search4EachItem
    self.itemProcess(item)
  File "bbs_a028.py", line 82, in itemProcess
    if not recheck_title(keyword,title):
NameError: global name 'keyword' is not defined
2013-12-26 10:13:48,254 - bbs_logger - ERROR - global name 'keyword' is not defined
Traceback (most recent call last):
  File "bbs_a028.py", line 92, in main
    obj.main()
  File "/home/lgy/Code/lgy_git/Python/newspider/BaseBBS.py", line 46, in main
    count=self.searchWrapper(count)
  File "/home/lgy/Code/lgy_git/Python/newspider/BaseBBS.py", line 66, in searchWrapper
    self.search4EachItem(items)
  File "/home/lgy/Code/lgy_git/Python/newspider/BaseBBS.py", line 99, in search4EachItem
    self.itemProcess(item)
  File "bbs_a028.py", line 83, in itemProcess
    if not recheck_title(keyword,title):
NameError: global name 'keyword' is not defined
2013-12-26 11:29:22,592 - bbs_logger - ERROR - search4EachItem() takes exactly 2 arguments (3 given)
Traceback (most recent call last):
  File "bbs_a028.py", line 107, in main
    obj.main()
  File "/home/lgy/Code/lgy_git/Python/newspider/BaseBBS.py", line 46, in main
    count=self.searchWrapper(count)
  File "bbs_a028.py", line 98, in searchWrapper
    self.search4EachItem(items,keyword)
TypeError: search4EachItem() takes exactly 2 arguments (3 given)
2013-12-26 11:30:37,302 - bbs_logger - ERROR - itemProcess() takes exactly 3 arguments (2 given)
Traceback (most recent call last):
  File "bbs_a028.py", line 111, in main
    obj.main()
  File "/home/lgy/Code/lgy_git/Python/newspider/BaseBBS.py", line 46, in main
    count=self.searchWrapper(count)
  File "bbs_a028.py", line 98, in searchWrapper
    self.search4EachItem(items,keyword)
  File "bbs_a028.py", line 107, in search4EachItem
    self.itemProcess(item)
TypeError: itemProcess() takes exactly 3 arguments (2 given)
